sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -441,6 +455,12 @@ public void run() {
             PreferencesStorage.remove(PREFERENCE_KEY_POSTPONE_TIME);
             PreferencesStorage.remove(PREFERENCE_KEY_UPDATE_SETUP_FAILED_PACKAGE_HASH_KEY);
             PreferencesStorage.remove(PREFERENCE_KEY_UPDATE_SETUP_FAILED_MESSAGE_KEY);
+
+            /* Disable the distribute info tracker if the feature is enabled. */
+            if (this.isDistributionGroupIdFeatureEnabled() && mDistributeInfoTracker != null) {
```

Comment:
Looks like checking `isDistributionGroupIdFeatureEnabled` is not necessary here.

Reply:
This PR is not user-facing so I've removed feature flag. This check removed as well. Thank you.